App.tsx

```
@@ -1,67 +1,62 @@
+const App = () => {
   const [userNumber, setUserNumber] = useState<number | null>(null);
   const [rounds, setRounds] = useState(0);
   const [dataLoaded, setDataLoaded] = useState(false);
 
+  useEffect(() => {
+    Font.loadAsync({
+      "open-sans": require("./assets/fonts/OpenSans-Regular.ttf"),
+      "open-sans-bold": require("./assets/fonts/OpenSans-Bold.ttf"),
+    })
+      .then(() => setDataLoaded(true))
+      .catch(error => console.log(error, "AppLoading error"));
```

ESLint

[error] Unexpected console statement.

(no-console)